migrate-from-mariadb.md

```diff
@@ -0,0 +1,334 @@
+---
+title: Migrate Data from MariaDB to TiDB
+summary: Learn how to migrate data from MariaDB to TiDB.
+---
+
+# Migrate Data from MariaDB to TiDB
+
+This document describes how to migrate data from a MariaDB server installation to a TiDB cluster.
+
+## Prerequisites
+
+- Install [Dumpling](/dumpling-overview.md) and [TiDB Lightning](/tidb-lightning/tidb-lightning-overview.md).
+- Set up [Data Migration (DM)](/dm/dm-overview.md)
+- Make sure you have the required privileges on the MariaDB server that are required for Dumpling to export data.
```

Comment:
Also link to a reference for "required privileges" 🤔

Reply:
Moved to L27